lxd/instance/drivers/driver_qemu_templates.go

```diff
@@ -93,193 +93,253 @@ func qemuBaseSections(architecture string) []cfgSection {
 		})
 }
 
-var qemuMemory = template.Must(template.New("qemuMemory").Parse(`
-# Memory
-[memory]
-size = "{{.memSizeBytes}}M"
-`))
+type qemuMemoryOpts struct {
+	memSizeMB int64
+}
 
-var qemuSerial = template.Must(template.New("qemuSerial").Parse(`
-# Virtual serial bus
-[device "dev-qemu_serial"]
-{{- if eq .bus "pci" "pcie"}}
-driver = "virtio-serial-pci"
-bus = "{{.devBus}}"
-addr = "{{.devAddr}}"
-{{- end}}
-{{if eq .bus "ccw" -}}
-driver = "virtio-serial-ccw"
-{{- end}}
-{{if .multifunction -}}
-multifunction = "on"
-{{- end }}
+func qemuMemorySections(opts *qemuMemoryOpts) []cfgSection {
+	return []cfgSection{{
+		name:    "device \"memory\"",
```

Comment:
You could use bsckticks here for the string to avoid having to escape the quotes btw